backend/connector_processor/connector_processor.py

```diff
@@ -132,6 +134,8 @@ def test_connectors(connector_id: str, credentials: dict[str, Any]) -> bool:
             test_result = connector_impl.test_credentials()
             logger.info(f"{connector_id} test result: {test_result}")
             return test_result
+        except UnstractFSConnectorException as e:
```

Comment:
@kirtimanmishrazipstack do we really need a new base class when we already have `ConnectorError`? Can you try re-using the same?

Reply:
@chandrasekharan-zipstack using connectorError, we can't specify from which db or fs we are getting the execption

Moreover the way we have handled connectorError is not proper. Entire stack trace is visible to the user which is not a good UI

Reply:
moreover all the fs connector exception can be clubbed as UnstractFSConnectorException

Reply:
I'd still suggest inheriting from `ConnectorError`. If its implementation is not proper, let's try to correct it